qa/migration-tests/src/test/java/io/camunda/it/migration/util/MigrationITExtension.java

```diff
@@ -129,8 +142,17 @@ private void upgrade(final Map<String, String> envOverrides) {
 
     awaitImportersFinished();
```

Comment:
💭 Do we actually need to wait for this? The migrators would also wait as far as I know. Or maybe not all of them?